src/SelfUpdateCommand.php

```diff
@@ -2,16 +2,13 @@
 
 namespace SelfUpdate;
 
-use Composer\Semver\VersionParser;
-use Composer\Semver\Semver;
-use Composer\Semver\Comparator;
+use JetBrains\PhpStorm\NoReturn;
```

Comment:
This is nice because it prevents static analysis tools from thinking the command doesn't return without having to include a dummy return statement. But I would understand if you don't want IDE-specific gunk in here.